go.mod

```diff
@@ -4,9 +4,9 @@ go 1.18
 
 require (
 	github.com/btcsuite/btcd v0.23.1
-	github.com/btcsuite/btcd/btcec/v2 v2.2.0
-	github.com/btcsuite/btcd/btcutil v1.1.1
-	github.com/btcsuite/btcd/btcutil/psbt v1.1.4
+	github.com/btcsuite/btcd/btcec/v2 v2.2.1
```

Comment:
This commit should disappear when we rebase the unannounced and zero conf channel PRs on top of the Taproot/MuSig2 one.

Reply:
Do you mean after we bump to `v0.15.1-beta` in `Taproot/MuSig2 `?

Reply:
Yes